index.js

```diff
@@ -1,4 +1,4 @@
-const Clownface = require('./lib/Clownface')
+const { Clownface } = require('./lib/Clownface.js')
```

Comment:
The only commonjs file is the index, which prevents the `default` export from breaking the public API. This way the usage in node js does no break